### ISSUE TITLE
Regarding DreamBooth Training

### ISSUE THREAD

**Issue by stalin18** [youyuanyi/variance-as-Catalyst#1]
Hi! Thanks for your great work!
I have a question about DreamBooth training. In the paper you mention that "We select 50 different identities from each dataset and use 4 reference images per identity as training data.", however, when I try to train DreamBooth model using only 4 reference images, I do not get good results. Though, when I use LoRA, 4 reference images suffice. I wonder what I am missing, are there specific hyperparameters (learning rate, number of iterations) that need special attention? Did you also face such problem? Along with the clarification, if possible, can you please also share the hyperparameters you used for training DreamBooth using 4 reference images? Thank you in advance.

**Comment by youyuanyi** [comment]
Thank you for your interest in our work!
Regarding your question about DreamBooth training, 4 reference images are generally sufficient for training a standard subject if the hyperparameters are tuned correctly. The issue you are facing might be due to the learning rate or the lack of regularization.
Here are the specific settings we recommend for the vanilla DreamBooth training with 4 images:
1. Learning Rate: It should be lower than what is typically used for LoRA. I suggest trying 2e-6 or 5e-7.
2. Training Steps: Set max_train_steps to around 1000.
3. Prior Preservation (Crucial): When using a small number of images, regularization is key to preventing overfitting. Please ensure you include --with_prior_preservation and set --prior_loss_weight=1.0.

On the other hand, if you are specifically testing the effectiveness of our proposed method, please note that the optimal parameters for our method are independent of the DreamBooth training hyperparameters. For the specific configurations regarding our approach, please refer to the settings in main.py in our repository.
